<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Start a new lecture</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(222, 234, 228);</string>
  </property>
  <widget class="QLabel" name="dialog_label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>200</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>11</pointsize>
     <weight>7</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(33, 82, 115);
font: 57 11pt &quot;Montserrat Medium&quot;;</string>
   </property>
   <property name="text">
    <string>Do you want to start a New Lecture ?</string>
   </property>
  </widget>
  <widget class="QLabel" name="c10">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>380</y>
     <width>211</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>rings1.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="c9">
   <property name="geometry">
    <rect>
     <x>-70</x>
     <y>-50</y>
     <width>201</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>rings2.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_btn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(47, 144, 145);
color: rgb(243, 243, 243);
 border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>250</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 234, 228);
border-color: rgb(47, 144, 145);
color: rgb(33, 82, 115);
border-width: 2px;
border-style:outset;
border-radius: 15px;
</string>
   </property>
   <property name="text">
    <string>CANCEL</string>
   </property>
  </widget>
  <zorder>c9</zorder>
  <zorder>c10</zorder>
  <zorder>dialog_label</zorder>
  <zorder>ok_btn</zorder>
  <zorder>cancel_btn</zorder>
 </widget>
 <resources>
  <include location="app_images/circle2.qrc"/>
  <include location="app_images/circle1.qrc"/>
 </resources>
 <connections/>
</ui>
